models/POP/shell_scripts/cesm2_1/CESM2_1_setup_pmo

```diff
@@ -0,0 +1,763 @@
+
+      echo "Copying DART template for ${FNAME} and changing instance."
+
+      ${COPY} $DARTROOT/models/POP/shell_scripts/$cesmtagmajor/user_$streamname*template ${FNAME}
+
+      sed s/NINST/$instance/g ${FNAME} >! out.$$
+      ${MOVE} out.$$ ${FNAME}
+
+   else
+      echo "DIED Looking for a DART stream txt template for ${FNAME}"
+      echo "DIED Looking for a DART stream txt template for ${FNAME}"
+      exit -3
+   endif
+
+end
+
+# ==============================================================================
+# Building the case.
+# ==============================================================================
+
+echo ''
+echo 'Building the case.'
+echo ''
+
+${BUILD_WRAPPER} ./case.build
+
+if ( $status != 0 ) then
+   echo "ERROR: Case could not be built."
+   exit -5
+endif
+
+# ==============================================================================
+# Checking the case.
+# ==============================================================================
+
+echo ""
+echo "Checking the case."
+echo ""
+echo "1) cd ${RUNDIR}"
+echo "   and check the compatibility between the namelists/pointer"
+echo "   files and the files that were staged."
+echo ""
+echo "2) cd ${CASEROOT}"
+echo ""
+echo "3) The case is initially configured to NOT INVOKE ANY DART CODE."
+echo "   When you are ready to generate synthetic observations, configure and execute"
+echo "   the ${CASEROOT}/CESM_DART_config script."
+echo ""
+echo "4) The very first CESM advance (i.e. CONTINUE_RUN=FALSE)"
+echo "   STOP_N must be longer than *AT LEAST 2 TIMES* the coupling"
+echo "   frequency between the atmosphere and ocean."
+echo "   If coupling once a day, the first advance MUST be at least 48 hours."
+echo "   If coupling 4 times a day, the first advance MUST be at least 12 hours."
+echo "   After that, STOP_N can be as short as a single coupling frequency."
+echo ""
+echo "5) Verify the contents of env_run.xml and submit the CESM job:"
+echo "   ./case.submit -M begin,end"
+echo ""
+echo "6) After the job has run, check to make sure it worked and that"
+echo "   a: POP is creating netCDF restart files,"
+echo "   b: the right restart files exist in the run directory,"
+echo "   c: (if you're running DART) the archive dart/hist directory has the DART output,"
+echo "   d: everything is working correctly ..."
+echo ""
+echo "7) To extend the run in $STOP_N '"$stop_option"' steps,"
+echo "   change the env_run.xml variables:"
+echo ""
+echo "   ./xmlchange CONTINUE_RUN=TRUE"
+echo "   ./xmlchange RESUBMIT=<number_of_cycles_to_run>"
+echo "   ./xmlchange STOP_N=$STOP_N"
+echo ""
+echo "   and"
+echo "   ./case.submit"
+echo ""
+echo "Check the streams listed in the streams text files.  If more or different"
+echo 'dates need to be added, then do this in the $CASEROOT/user_*files*'
+echo "then invoke './preview_namelists' so you can check the information in the"
+echo "${CASEROOT}/CaseDocs or ${RUNDIR} directories."
+echo ""
```

Comment:
We ultimately decided that creating a 'here' document with these instructions and then cat'ting them out was a good idea. The document with the instructions then persists in the CASEROOT directory.
See cam-fv/shell_scripts/cesm2_1/setup_advanced
If you don't want to do that - its fine as it stands. Just a suggestion.

Reply:
Actually ... you do it in the CESM_DART_config ...